app/controllers/build_parts_controller.rb

```diff
@@ -6,6 +6,12 @@ class BuildPartsController < ApplicationController
   }
 
   def show
+    respond_to do |format|
+      format.html
+      format.json do
+        render :json => @build_part, include: { build_attempts: { include: :build_artifacts } }
```

Comment:
are you sure you need the build artifacts? Also, if you do need them, the way this is written will not include a url to view the actual artifact file.